src/main/java/access/Application.java

```diff
@@ -68,6 +73,11 @@ protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
 	}
 
 	@Bean
+	public Queue updateJobsQueue() {
+		return new Queue(String.format(JobMessageFactory.KAFKA_TOPIC_TEMPLATE, JobMessageFactory.UPDATE_JOB_TOPIC_NAME, SPACE));
```

Comment:
consider renaming this topic template to not contain the word kafka, since it's not specific to kafka.